statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -3225,11 +3229,16 @@ def test_serial_correlation(self, method, lags=None):
                     FutureWarning
                 )
 
+            if df_adjust:
+                df_model = self.df_model - 1
```

Comment:
can you call this model_df to align with name in function.

Reply:
Done